.github/workflows/pr-unit-tests.yaml

```
@@ -13,6 +13,7 @@ env:
 on:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
+  merge_group: { }
```

```suggestion
  merge_group:
    types: [checks_requested]
```

per https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#merge_group